final_project/dags/sql/user_profiles_transfer_from_data_lake_raw_to_silver.sql

```diff
@@ -0,0 +1,21 @@
+MERGE `de2022-stanislav-rezen.silver.user_profiles` main
+    USING user_profiles_jsonl as stage
+    ON main.email = stage.email
+    WHEN MATCHED THEN
+    UPDATE SET
+    full_name = stage.full_name,
+    state = stage.state,
+    birth_date = CAST(stage.birth_date as DATE),
+    phone_number = stage.phone_number
```

Comment:
When you update, you also should update logical_date and  start_date technical fields. 

Just FYI: The 3 technical fields I proposed work not so well for such cases because we do not use partitions. There additional technical fields can be added to track update history.